web-platform/website/src/pages/login.tsx

```diff
@@ -34,7 +34,9 @@ const LoginPage: NextPage = () => {
       router.replace("/get-extension");
       return null;
     }
-  } else if (user) {
+  }
+
+  if (user) {
```

Comment:
just want to double check that this logic change is intended

Reply:
Yes, it was.